src/preact/base-element.js

```diff
@@ -612,43 +573,6 @@ export class PreactBaseElement extends BaseElement {
     }
   }
 
-  /**
-   * Default handler for onLoad event
-   * Displays loader. Override to customize.
-   * @protected
-   */
-  handleOnLoad() {
```

Comment:
FMI: Do bento WCs not support loading/fallback/placeholder?

Reply:
My understanding is that fallbacks and placeholders are an AMP concept and we're leaving the job of toggling an element on or off to consumers

@alanorozco can you confirm?